backend/beetleblog-service/src/main/java/com/beetleblog/app/domains/User.java

```diff
@@ -0,0 +1,21 @@
+package com.beetleblog.app.domains;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@ToString
+public class User {
```

Comment:
We need to have other **Lombok** annotations here - `@EqualsAndHashCode`, `@NoArgsConstructor` and `AllArgsConstructor`,

Reply:
Done